code/game/objects/items/devices/binoculars.dm

```diff
@@ -56,6 +57,9 @@
 	var/last_x = "UNKNOWN"
 	var/last_y = "UNKNOWN"
 
+	/// Normally used for the small green dot signifying coordinations-obtaining mode.
+	var/range_laser_overlay = "laser_range"
```

Comment:
why are these vars not defines? dont need to be vars if never changed

Reply:
to allow them to be changed in the future, like if someone adds weird-shaped binoculars for CLF or UPP or whatever. it's not a performance concern because binos are rare